lib/cisco_node_utils/bgp.rb

```diff
@@ -786,10 +786,20 @@ def default_reconnect_interval
     # route_distinguisher
     # Note that this property is supported by both bgp and vrf providers.
     def route_distinguisher
+      if @vrf.nil? || @vrf == 'default'
+        fail Cisco::UnsupportedError.new('bgp', 'route_distinguisher', 'get',
```

Comment:
Logic was right for this method before. Getter and default should return nil if unsupported - only setter raises an exception.